autogpt_platform/frontend/src/components/admin/marketplace/expandable-row.tsx

```diff
@@ -0,0 +1,209 @@
+          {listing.creator_email || "Unknown"}
+        </TableCell>
+        <TableCell onClick={() => setExpanded(!expanded)}>
+          {latestVersion?.sub_heading || "No description"}
+        </TableCell>
+        <TableCell onClick={() => setExpanded(!expanded)}>
+          {latestVersion?.status && getStatusBadge(latestVersion.status)}
+        </TableCell>
+        <TableCell onClick={() => setExpanded(!expanded)}>
+          {latestVersion?.date_submitted
+            ? formatDistanceToNow(new Date(latestVersion.date_submitted), {
+                addSuffix: true,
+              })
+            : "Unknown"}
+        </TableCell>
+        <TableCell className="text-right">
+          <div className="flex justify-end gap-2">
+            <Button size="sm" variant="outline">
+              <ExternalLink className="mr-2 h-4 w-4" />
+              Builder
+            </Button>
+
+            {latestVersion?.status === SubmissionStatus.PENDING && (
+              <ApproveRejectButtons version={latestVersion} />
+            )}
+          </div>
+        </TableCell>
+      </TableRow>
+
+      {/* Expanded version history */}
+      {expanded && (
+        <TableRow>
+          <TableCell colSpan={7} className="border-t-0 p-0">
+            <div className="bg-muted/30 px-4 py-3">
+              <h4 className="mb-2 text-sm font-semibold">Version History</h4>
+              <Table>
+                <TableHeader>
+                  <TableRow>
+                    <TableHead>Version</TableHead>
+                    <TableHead>Status</TableHead>
+                    {/* <TableHead>Changes</TableHead> */}
+                    <TableHead>Submitted</TableHead>
+                    <TableHead>Reviewed</TableHead>
+                    <TableHead>External Comments</TableHead>
+                    <TableHead>Internal Comments</TableHead>
+                    <TableHead>Name</TableHead>
+                    <TableHead>Sub Heading</TableHead>
+                    <TableHead>Description</TableHead>
+                    {/* <TableHead>Categories</TableHead> */}
+                    <TableHead className="text-right">Actions</TableHead>
+                  </TableRow>
+                </TableHeader>
+                <TableBody>
+                  {listing.versions
+                    .sort((a, b) => (b.version ?? 1) - (a.version ?? 0))
+                    .map((version) => (
+                      <TableRow key={version.store_listing_version_id}>
+                        <TableCell>
+                          v{version.version || "?"}
+                          {version.store_listing_version_id ===
+                            listing.active_version_id && (
+                            <Badge className="ml-2 bg-blue-500">Active</Badge>
+                          )}
+                        </TableCell>
+                        <TableCell>{getStatusBadge(version.status)}</TableCell>
+                        {/* <TableCell>
```

Comment:
Remove commented cell here and the table head one above and below ?

Reply:
so this works on the admin side but not the user submission side. Its kinda weird. WDYT? should I uncomment (and show "") or remove and add it back later?

Reply:
the user field to input changes_summary doesnt exist yet

Reply:
or the backend code for the user side is just not updated yet? 
if so, then it doesn't matter I think, we will fix it right after this ?

Reply:
I see then then it's fine :+1: